tapdb/asset_minting_test.go

```diff
@@ -421,17 +422,18 @@ func seedlingsToAssetRoot(t *testing.T, genesisPoint wire.OutPoint,
 		)
 		require.NoError(t, err)
 
-		// Finally make a new asset commitment (the inner SMT tree) for
-		// this newly created asset.
-		assetRoot, err := commitment.NewAssetCommitment(
-			newAsset,
-		)
-		require.NoError(t, err)
+		if groupKey != nil {
+			engine, err := vm.New(newAsset, nil, nil)
```

Comment:
Hmm perhaps the unit tests on the higher level are enough? Feels strange to run VM checks at the DB abstraction level.

Reply:
Agree, though this mirrors the sprouting step in the caretaker which also now runs the VM. Though we don't make interesting witnesses here so could remove.

Reply:
Removed in latest version.